backend/event-parse/parse.go

```diff
@@ -368,7 +388,7 @@ func (s *Snapshot) InjectStylesheets() error {
 		if !ok || !strings.Contains(href, "css") {
 			continue
 		}
-		data, err := fetch.fetchStylesheetData(href)
+		data, err := fetch.fetchStylesheetData(href, s)
 		if err != nil {
 			continue
```

Comment:
would it be helpful to add some backend logging here if the request fails? now that there are more error cases here